packages/slice-machine/pages/changes.tsx

```diff
@@ -26,6 +33,12 @@ const changes: React.FunctionComponent = () => {
     if (!isOnline) {
       return <OfflinePage />;
     }
+    if (
+      authStatus === AuthStatus.UNAUTHORIZED ||
+      authStatus === AuthStatus.FORBIDDEN
+    ) {
+      return <AuthErrorPage />;
```

Comment:
think we can pass the authStatus to the `<AuthErrorPage />` component in order to give more info to the user:
- AuthStatus.UNAUTHORIZED => tell the user he needs to login and display a login button
-  AuthStatus.FORBIDDEN => tell the user he does not have access to the repo and do not display the login button

unclear from the design so need to check with Come and/or Louis

Reply:
defiantly check with them :)

Reply:
send them a message